### PR TITLE
Improve look and feel of olm site

### PR DESCRIPTION
**Pull request by anmol372**
Changes:
1. Added styling imports from themes/docsy
2. Corrected font family, sizing, and letter-spacing for docs
3. Increased size of headings on the site to make them larger than their subtexts.

### CODE REVIEW

File: content/en/docs/getting-started/_index.md [hunk 1]
```diff
@@ -17,38 +17,37 @@ These pages are under construction. TODO: Check Prerequisites and update
...
 
 ### Install Released OLM
-For installing release versions of OLM, for example version 0.12.0, you can use the following command:
+For installing release versions of OLM, for example version 0.15.1, you can use the following command:
 
-```bash
-export olm_release=0.15.1
-kubectl apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/crds.yaml
-kubectl apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/olm.yaml
+```sh
```
Comment (estroz) [inline content/en/docs/getting-started/_index.md:29]: If we expect users to copy these shell commands verbatim then I suggest we do not prepend `$ ` to each line of a `sh` block; this prefix is only visually useful when using a `console` block with command output as an example.

Comment (anmol372) [reply content/en/docs/getting-started/_index.md:29]: That makes sense, I prepended the `$ ` prefix to match the style on the SDK website, but I have corrected it.

File: themes/docsy/layouts/partials/section-index.html [hunk 2]
```diff
@@ -18,9 +18,9 @@
         {{ range $pages }}
             {{ if eq .Parent $parent }}
                 <div class="entry">
-                    <h5>
+                    <h3>
```
Comment (anik120) [inline themes/docsy/layouts/partials/section-index.html:21]: This folder is a sub-module(i,e vendored), and therefore should not be changed. If we want to make a change to any of these files, we need to copy them in the root folder and make these changes. 

For example, this particular file needs to be copied to the [partials](https://github.com/operator-framework/olm-docs/tree/master/layouts/partials) folder in our root to make these changes.

File: themes/docsy/assets/scss/section-index.scss [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
-//     .entry {
-//         padding: .75rem;
-//     }
+    .entry {
+        padding: .75rem;
+    }
 
-//     h5 {
-//         margin-bottom: 0;
+    h5 {
```
Comment (anik120) [inline themes/docsy/assets/scss/section-index.scss:7]: Similar comment as below. We shouldn't be making changes to the `themes` submodule.

File: layouts/docs/single.html [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{{ define "main" }}
+{{ .Render "content" }}
+{{ end }}
```
Comment (anik120) [inline layouts/docs/single.html:3]: nit: please add a new line to this file

**Review by anik120 — CHANGES_REQUESTED** [review]
@anmol372 thank you so much for this PR! I've left a few comments for you.

File: themes/docsy/layouts/partials/section-index.html (hunk 2 above)
Comment (anmol372) [reply themes/docsy/layouts/partials/section-index.html]: I have undone the changes to vendor files and created a new file in the layouts/partials directory.

File: layouts/docs/single.html (hunk 4 above)
Comment (anmol372) [reply layouts/docs/single.html]: Done

File: assets/scss/_docs.scss [hunk 5]
```diff
@@ -17,7 +17,8 @@
     h1,h2,h3,h4,h5,h6 {
         color: var(--of--Color--brand--200);
         font-weight: 800;
-        margin: var(--of--spacer--md) 0 var(--of--spacer--sm);
+        letter-spacing: 2px;
```
Comment (anik120) [inline assets/scss/_docs.scss:20]: @anmol372 could we revert this change? We specifically wanted ` margin: var(--of--spacer--md) 0 var(--of--spacer--sm);` because the spacing between the heading was looking really bad

**Comment by anik120** [comment]
@anmol372 could you squash your commits please

**Comment by anik120** [comment]
@anmol372 could you rebase your changes off the latest master please

File: assets/scss/_docs.scss (hunk 5 above)
Comment (anmol372) [reply assets/scss/_docs.scss:20]: Done

**Review by anik120 — APPROVED** [review]
/lgtm
